packages/codemirror/style/base.css

```diff
@@ -231,3 +231,14 @@
 .cm-s-jupyter .CodeMirror-gutter {
   background-color: var(--jp-layout-color2);
 }
+
+.cm-s-jupyter .CodeMirror-matchingbracket {
+  color: var(--jp-mirror-editor-matchingbracket);
+}
+.cm-s-jupyter .CodeMirror-nonmatchingbracket {
+  color: var(--jp-mirror-editor-nonmatchingbracket);
+}
```

Comment:
I guess that (surprisingly), we weren't able to customize these before. Nice improvement!